docs/docs/getting-started/first-steps.md

```diff
@@ -164,6 +164,35 @@ If you don't have a cloud subscription, check out [MiniConstellation](first-step
 
         Supported are all machines from the N2D family. It defaults to `n2d-standard-4` (4 vCPUs, 16 GB RAM), but you can use any other VMs from the same family. Refer to [N2D machine series](https://cloud.google.com/compute/docs/general-purpose-machines#n2d_machines) or run `constellation config instance-types` to get the list of all supported options.
 
+    </tabItem>
+    <tabItem value="aws" label="AWS">
+
+    * **region**: The name of your chosen AWS data center region, e.g., `us-east-2`.
+
+        TODO: What is the list of supported regions for our images?
```

Comment:
@malt3 Is [this](https://github.com/edgelesssys/constellation/blob/feat/docs/aws-install/.github/workflows/build-os-image.yml#L270-L271) the full list of regions we target for release?

Reply:
Yes, we decided on having one image in every major region. Of course images from us-east-2 cannot be used in us-east-1, but I hope our users are flexible here. Supporting more location will be a pain, since we probably want to clean up old images at some point and we would have to do that in every region then.

Reply:
We should add a disclaimer here (and maybe in the code if we see no AMI for the region) that users can open an issue to get the image replicated into their region.

Reply:
I can create an issue template that we can link to for that purpose

Reply:
I re-used the feature template with a prefilled title, feel free to improve this as you see fit :)